src/medCore/gui/viewContainers/medViewContainer.cpp

```diff
@@ -168,6 +172,13 @@ medViewContainer::medViewContainer(medViewContainerSplitter *parent): QFrame(par
     connect(d->maximizedAction, SIGNAL(triggered()), this, SLOT(toggleMaximized()));
     d->maximizedAction->setEnabled(false);
 
+    d->saveSceneAction = new QAction(tr("Save scene"), d->toolBarMenu);
+    d->saveSceneAction->setToolTip(tr("Save container content as is."));
+    d->saveSceneAction->setIcon(QIcon(":icons/saveScene.png"));
```

Comment:
As i said on irc last week, i do not like very much this kind of button. Music/medInria has really few icon buttons, and usualy when we use an icon button it's smaller than this one. It would be more consistent with a regular "Save scene" button i think. As far as i remember, it was needed also the put the button at left also: it looked a bit lost in an empty area.